ci/pipelines/deploy-to-production.yml

```diff
@@ -1188,6 +1322,23 @@ jobs:
           image: frontend-ecr-registry-prod/image.tar
           additional_tags: parse-perf-release-tag/tag
 
+  - name: retag-nginx-forward-proxy-image-for-test-perf
+    plan:
+      - get: pay-ci
+      - get: nginx-forward-proxy-ecr-registry-prod
+        params:
+          format: oci
+        passed: [deploy-frontend-to-prod]
```

Comment:
Do we want to use the `smoke-test-frontend-on-prod` as the check here?

Reply:
Possibly. At the moment this task is under the [nginx-forward-proxy group](https://github.com/alphagov/pay-ci/blob/master/ci/pipelines/deploy-to-production.yml#L483-L485) which doesn't have the `smoke-test-frontend-on-prod` task. 
It seems like we're using a deploy to frontend to test any new nginx-forward-proxy images.

Reply:
Sorry, I might not have been clear. It's not possible to use the `smoke-test-frontend-on-prod` as the check here, unless we move `retag-nginx-forward-proxy-image-for-test-perf` to the [frontend group](https://github.com/alphagov/pay-ci/blob/master/ci/pipelines/deploy-to-production.yml#L429-L434).

Reply:
Ah I see. OK - let's leave as is then.